pages/services/prometheus/0.1.0-2.3.1/install/install.md

```diff
@@ -0,0 +1,299 @@
+    To federate data from promethes slave server to global prometheus,slave prometheus server endpoints to be passed as target into global prometheus server under prometheus configuration yml.
+      
+## Prerequisites
+   
+- If you are using Enterprise DC/OS, you may [need to provision a service account](https://docs.mesosphere.com/1.10/security/ent/service-auth/custom-service-auth/) before installing DC/OS Prometheus Service. Only someone with `superuser` permission can create the service account.
+  - `strict` [security mode](https://docs.mesosphere.com/1.10/security/ent/service-auth/custom-service-auth/) requires a service account.
+  - In `permissive` security mode a service account is optional.
+  - `disabled` security mode does not require a service account.
+- Your cluster must have at least 3 private nodes.
+
+# Installing from the DC/OS CLI
+
+To start a basic test cluster of Prometheus, run the following command on the DC/OS CLI. Enterprise DC/OS users must follow additional instructions.
+
+   ```shell
+   dcos package install prometheus 
+   ```
+
+This command creates a new instance with the default name prometheus. Two instances cannot share the same name, so installing additional instances beyond the default instance requires customizing the name at install time for each additional instance. However, the application can be installed using the same name in case of foldered installation, weherein we can install the same application in different folders.
+
+All dcos prometheus CLI commands have a --name  argument allowing the user to specify which instance to query. If you do not specify a service name, the CLI assumes a default value matching the package name, i.e. prometheus. The default value for --name can be customized via the DC/OS CLI configuration:
+
+   ```shell
+   dcos prometheus --name=prometheus <cmd>
+   ```
+
+You can specify a custom configuration in an `options.json` file and pass it to `dcos package install` using the `--options` parameter.
+
+   ```shell
+   dcos package install prometheus --options=options.json
+   ```
+
+For more information on building the `options.json` file, see [DC/OS documentation](https://docs.mesosphere.com/latest/usage/managing-services/config-universe-service/) for service configuration access.
+
+## Installing from the DC/OS Web Interface
+
+Note:  Alternatively, you can install Prometheus from the DC/OS web interface by clicking on Deploy after selecting the app from Catalog.
+   
+If you install Prometheus from the DC/OS web interface, the 
+dcos prometheus CLI commands are not automatically installed to your workstation. They may be manually installed using the DC/OS CLI:
+
+
+   ```shell
+   dcos package install prometheus --cli
+   ```
+
+## Note : For Prometheus configuration user must be aware of all the required configuration to be mentioned under prometheus.yml
```

Comment:
Should document the defaults in the framework, what they do and sections in prometheus.yml which should not be changed.

Reply:
Updated,building locally and will raise new PR , post making all the change